integration-tests/tests/bombastic.rs

```diff
@@ -174,6 +175,46 @@ async fn bombastic_search(context: &mut BombasticContext) {
     })
     .await;
 
+    // Update and check reindex
```

Comment:
This test is getting a little big, no? Can we affirm the re-indexing in its own test that uploads an sbom twice, but doesn't do much else other than compare the timestamps?

Reply:
Agreed, I will move this and the 'invalid search' test cases into separate tests.